color_detector.py

```diff
@@ -1,30 +1,40 @@
 import cv2
-from numpy import array
+import numpy as np
 
 
-class ColorDetector:
+RED_LOWER = np.array([0, 50, 140])
+RED_UPPER = np.array([200, 150, 255])
 
-    BLUE_LOWER = array([70, 50, 50])
-    BLUE_UPPER = array([120, 255, 255])
+BLUE_LOWER = np.array([0, 0, 0])
+BLUE_UPPER = np.array([255, 255, 110])
 
-    RED_LOWER = array([120, 50, 50])
-    RED_UPPER = array([200, 255, 255])
 
-    def __init__(self, image):
-        self.image = image
+def detect(image, color, return_all=False):
 
-    def detect(self, color):
+    # HoughCircles code
+    gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+    blur = cv2.medianBlur(gray, 9)
+    circles = cv2.HoughCircles(blur, cv2.HOUGH_GRADIENT, dp=1, minDist=120, param1=150, param2=30, minRadius=85)
```

Comment:
Is this up to date? I thought in the meeting you said we're not going to use HoughCircles moving forward.